packages/ui/react-ui-editor/src/extensions/comments.ts

```diff
@@ -151,6 +155,65 @@ export type CommentsOptions = {
   onHover?: (el: Element, shortcut: string) => void;
 };
 
+const trackPastedComments = (onMove: (threadID: string, cursor: string) => void) => {
```

Comment:
exposing onCut would be useful to allow the app to mark comments that are to be potentially deleted.

Reply:
Aren't those just the comments that cover an empty range?